content/docs/how_we_work/defining-done.md

```diff
@@ -0,0 +1,86 @@
+---
+# Page title as it appears in the navigation menu
+title: "Defining Project Doneness"
+# Adjust the weight to reorder menu items (lower numbers appear first)
+weight: 2
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Project Doneness
+
+> This document was produced as part of Accelerating AI in the Arts and Humanities (AAAH) produces outputs of professional quality for the research community.
+
+> Assuming you're starting with a code base that already exists, this document is designed to help you discuss scope of RSE involvement
+> with collaborators and define when a project is done.
+> This means making it easier for your current and future research, accelerating the impact and legacy of the software,
+> expanding communities of practice, and providing a confident codebase for future research.
+> This framework highlights what a maximal 'definition of done' would look like for mutual success to deliver reproducible and
+> credible code for all. It is expected that not all engagements will hit the maximum extents, but rather this is to be negotiated
+> and shaped by the resources available.
+
+>For each project discuss which "Level" in each of the following categories is a MINIMUM aim for defining done:
```

Comment:
```suggestion
This document was produced as part of Accelerating AI in the Arts and Humanities (AAAH) produces outputs of professional quality for the research community.

Assuming you're starting with a code base that already exists, this document is designed to help you discuss scope of RSE involvement
with collaborators and define when a project is done.
This means making it easier for your current and future research, accelerating the impact and legacy of the software,
expanding communities of practice, and providing a confident codebase for future research.
This framework highlights what a maximal 'definition of done' would look like for mutual success to deliver reproducible and
credible code for all. It is expected that not all engagements will hit the maximum extents, but rather this is to be negotiated
and shaped by the resources available.

For each project discuss which "Level" in each of the following categories is a MINIMUM aim for defining done:
```

Reply:
Thanks, I made those changes. I agree definitions look better but they didn't play well with the subtitle markup. i've added `&nbsp;` as a hack solution... 😅